pyth/client/src/client.rs

```diff
@@ -132,6 +132,9 @@ impl PythClientTrait for PythClient {
                                 },
                             }
                         }
+                        else {
+                            error!("Pyth SSE connection close");
+                        }
```

Comment:
This part isn't well formatted. rustfmt doesn't format contents inside macros (`stream! { ... }` here) so make sure to format them manually.